dist/tmpfiles.d/zincati.conf

```diff
@@ -12,3 +12,9 @@ d     /run/zincati/public    0775 zincati zincati  -   -
 
 # Legacy symlink to metrics socket
 L+    /run/zincati/private/metrics.promsock  - - - - ../public/metrics.promsock
+
+# Runtime MOTD messages which Zincati can write to
+d     /run/zincati/public/motd.d  0775 zincati zincati  -   -
+
+# Symlink to deadend MOTD message
+L+    /run/motd.d/85-zincati-deadend.motd  - - - - ../zincati/public/motd.d/deadend.motd
```

Comment:
The target here won't exist until the first successful graph fetch (and possibly never, in case of failures or if Zincati is disabled).

Should we start with a `/run/motd.d/85-zincati-deadend.motd -> /dev/null` and then update the symlink on each graph refresh maybe?

Reply:
I had missed thinking about the initial state of the symlink (again) - yes, we should avoid the dangling symlink. Linking to `/dev/null` would be good, however, runs into a problem with the `zincati` user getting a `Permission denied` when removing and re-creating the symlink with the new location. AFAIK, there isn't a way to overwrite the symlink target without deleting/recreating the symlink file (until https://lore.kernel.org/linux-fsdevel/cover.1524549513.git.osandov@fb.com/ gets accepted).

As an alternative, could we create the destination file upfront via tmpfiles.d, initializing it as an empty file? I updated the tmpfiles.d config for this.

Reply:
Ah, I forgot that we cannot atomically update an existing symlink. Yes, the tmpfiles way seems ok.